### PR TITLE
updated UpdateLEDStates method, Update method, and DriverFeedback constructor to support LEDs

### PR DESCRIPTION
**Pull request by JamesLimothy**
still have to add config for leds in code generator

### CODE REVIEW

File: src/main/cpp/configs/RobotConfig.cpp [hunk 1]
```diff
@@ -89,6 +89,10 @@ void RobotConfig::DefineMechanisms()
 {
 }
 
+void RobotConfig::DefineLEDs()
+{
+}
```
Comment (Tan560) [inline src/main/cpp/configs/RobotConfig.cpp:94]: @muscatd75 Should we be changing RobotConfig.cpp and .h? Isn't this generated

File: src/main/cpp/configs/RobotConfigpracticeBot_9999.cpp [hunk 2]
```diff
@@ -49,6 +50,11 @@ void RobotConfigpracticeBot_9999::DefineMechanisms()
 
 }
 
+void RobotConfigpracticeBot_9999::DefineLEDs()
+{
+	DragonLeds::GetInstance()->Initialize(7,50);
```
Comment (Tan560) [inline src/main/cpp/configs/RobotConfigpracticeBot_9999.cpp:55]: Same comment as above @muscatd75

**Review by Tan560 — CHANGES_REQUESTED** [review]
Can we test this on the robot before commiting to main please. Thank you

File: src/main/cpp/configs/RobotConfig.cpp (hunk 1 above)
Comment (muscatd75) [reply src/main/cpp/configs/RobotConfig.cpp:94]: Yes, this is the default function and can be overridden by the specific robots

File: src/main/cpp/configs/RobotConfigpracticeBot_9999.cpp (hunk 2 above)
Comment (muscatd75) [reply src/main/cpp/configs/RobotConfigpracticeBot_9999.cpp:55]: This will be generated

**Comment by witcpalek** [comment]
In @muscatd75 pull request

**Pull request closed without merging**